installers/ansible/roles/pgo-operator/tasks/namespace.yml

```
@@ -28,7 +28,7 @@
   with_items: "{{ nslist.split(',') | map('trim') | list }}"
   when:
     - namespace_mode != 'dynamic'
-    - reconcile_rbac == 'true'
+    - not (reconcile_rbac | bool)
```

Seems like here and below should lack `not`. Anything that used to be `reconcile_rbac == 'true'`

```suggestion
    - reconcile_rbac | bool
```